aten/src/ATen/native/Sorting.cpp

```diff
@@ -118,6 +119,16 @@ void quantile_impl(
   TORCH_CHECK(
       self.device() == out.device(),
       "quantile() out tensor must be on the same device as the input tensor");
+  const std::vector<string> interpolations{
```

Comment:
I'm not sure we want to pass strings to operator kernels (in cases like loss reduction functions to pick an enum in python).  Not a big deal for this PR.

I would recommend we canonicalize the string into an enum as in https://github.com/pytorch/pytorch/blob/1833009202ee5b8e4e25affc325dec87cad87d36/aten/src/ATen/native/TensorAdvancedIndexing.cpp#L768-L779 -- it will be easier to change in the future and you don't have to parse it multiple times.